Guideline.md

```diff
@@ -282,6 +281,12 @@ At the event, make sure there are an appropriate number of sober staff on hand t
 #### 9.1 Name badges
 The purpose of a name badge is to identify people easily. As such, it should be easy to read! You should use a large, easy to read font. Sans serif fonts are generally regarded as a good choice. Make the name stand out and contrast well - avoid grey on grey! A font size of 18 points or bigger if you can make it fit would make a difference.
 
+Consider allowing people to define or even write in their prefered name, rather than simply printing their legal name and do not require photo ID but rather use a QR code send by e-mail to allow people to identify themselves. Furthermore allow space for people to define their preferred pronouns or [provide premade buttons/stickers](http://yupgup.com/pronouns.html).
+
+It may make sense to provide stickers for people to denote things like "first time attendee", "ask me about [ - space to write in- ]", "talk to me" to foster more interaction between attendees.
+
+Providing stickers or different lanyards (colors and patterns) to enable people to choose if they want photos taken of themselves and how these may be used. For inspiration see [here](https://twitter.com/imeluny/status/956072960296673280?lang=en).
```

Comment:
Make sure that these identifying features are large enough to be seen post-event while processing photos for publication.

Reply:
thx again!